File: Ritika kumar/Basicsofprogramming/Reverse_a_number.java
public class Reverse_a_number
{
    public static void main(String[]args)
    {
        int n=457;
        int temp =n;
        while(temp!=0)
        {
            int x =temp%10;
            System.out.print(x);
            temp=temp/10;
        }
    }
}

File: Kritika Tripathi/Basics Of Programming/Getting Started/reverseanumber.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    cin >> n;

    //write your code here
    int rem, temp = n;
    while(temp!=0){
        rem = temp%10;
        cout<<rem<<endl;
        temp/=10;
    }
    return 0;
}